deno/src/bot.ts

```diff
@@ -276,28 +276,23 @@ export class Bot<C extends Context = Context> extends Composer<C> {
             // fetch updates
             const offset = this.lastTriedUpdateId + 1
             let updates: Update[] | undefined = undefined
-            let maxRetries = 1200 // 1 hour of continuously failing requests
             do {
                 try {
                     updates = await this.api.getUpdates(
                         { offset, limit, timeout, allowed_updates },
                         this.pollingAbortController.signal
                     )
                 } catch (error) {
-                    if (this.pollingRunning && error instanceof GrammyError) {
+                    if (this.pollingRunning) {
                         debugErr(
-                            `Call to \`getUpdates\` failed, retrying in 3 seconds ...`
+                            'Call to `getUpdates` failed, retrying in 3 seconds ...'
                         )
                         await new Promise(r => setTimeout(r, 3000))
                     } else {
                         throw error
                     }
                 }
-            } while (
-                updates === undefined &&
-                this.pollingRunning &&
-                maxRetries-- > 0
-            )
+            } while (updates === undefined && this.pollingRunning)
```

Comment:
This condition could be further simplified to
```suggestion
            } while (this.pollingRunning)
```

if you add `break` right after `updates = await this.api.getUpdates(`

Reply:
That true, I just try to avoid `break` and `continue` as much as possible as they're basically `goto` …